src/python/grpcio/grpc/__init__.py

```diff
@@ -31,12 +31,33 @@
 
 import abc
 import enum
+import sys
 
 import six
 
 from grpc._cython import cygrpc as _cygrpc
 
 
+############################### Extension Shims ################################
+
+try:
```

Comment:
Add a comment to the effect of "these are here to maintain compatibility; do not use!"?